vogue/load/sample.py

```diff
@@ -0,0 +1,22 @@
+from vogue.load.lims import build_sample
+
+from genologics.lims import Lims
+from genologics.entities import Sample
+from genologics.config import BASEURI,USERNAME,PASSWORD
+lims = Lims(BASEURI,USERNAME,PASSWORD)
+
+
+def load_lims_sample(adapter, lims_id):
+    lims_sample = Sample(lims, id = lims_id)
+    mongo_sample = build_sample(lims_sample)
+    adapter.add_or_update_sample(mongo_sample)
+
+def dry_run(adapter, lims_id):
+    lims_sample = Sample(lims, id = lims_id)
+    mongo_sample = build_sample(lims_sample)
+    print(mongo_sample)
```

Comment:
```suggestion
    LOG.info(mongo_sample)
```
Perhaps something more informative here. That it is a dry run etc